tests/compat.sh

```diff
@@ -39,10 +39,21 @@ SRVMEM=0
 # default commands, can be overridden by the environment
 : ${M_SRV:=../programs/ssl/ssl_server2}
 : ${M_CLI:=../programs/ssl/ssl_client2}
-: ${OPENSSL_CMD:=openssl} # OPENSSL would conflict with the build system
+: ${OPENSSL:=openssl} # OPENSSL would conflict with the build system
```

Comment:
Now that we are using OPENSSL everywhere, shouldn't this comment be removed? I'm assuming the comment was originally written to explain why `OPENSSL_CMD` was used instead of `OPENSSL`.

Reply:
Absolutely, I removed the comment on the similar line in `ssl-opt.sh` but forgot this one. Thanks for noticing!